app/controllers/users_controller.rb

```diff
@@ -0,0 +1,19 @@
+class UsersController < ApplicationController
+  before_action :find_user, only: [:show]
+
+  def index
+    @users = User.all
+  end
+
+  def show; end
+
+  def new; end
+
+  def find_user
+    @user = User.find(params[:id])
+  rescue ActiveRecord::RecordNotFound
```

Comment:
* As stated in the [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project_v1_1.md#:~:text=NOT%20use%20rails%20generate%20scaffold%20command.%20The%20point%20of%20this%20project%20is%20to%20show%20you%20how%20to%20set%20up%20controllers%20by%20yourself.) NOT to use rails generate scaffold command. The point of this project is to show you how to set up controllers by yourself. Only create the contorllers and actions that you need For example 👇 
```
# app/controllers/users_controller.rb
class UsersController < ApplicationController
  
  def index; end

  def show; end

end

# app/controllers/posts_controller.rb
class PostsController < ApplicationController
  
  def index; end

  def show; end

end
```